.github/workflows/build-check.yml

```diff
@@ -7,16 +7,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [ '12', '14' ]
+        node: [ '15' ]
     name: Node ${{ matrix.node }}
     steps:
       - uses: actions/checkout@v2
       - name: Setup node
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node }}
-      - name: Install npm@6
-        run: npm i npm@6 # to use workspace
+      - name: Install latest npm
+        run: npm i npm@7
```

Comment:
actually we can remove this task

Reply:
oh yeah since it will install 7 anyway, will remove it